modules/es-extensions/store/extensions/app/greg-diff/pages/diff-view.jag

```diff
@@ -5,15 +5,16 @@ require('/modules/store.js').exec(function (ctx) {
     caramel = require('caramel');
     var ui = require('rxt').ui;
     var tenantApi = require('/modules/tenant-api.js').api;
-
+    var tenantDetails = tenantApi.createTenantDetails(request, session);
+    var tenantId = tenantDetails.tenantId;
     var options = ref.getQueryOptions(ctx.request.getQueryString());
     if (!options.type) {
         ctx.response.sendError(404, 'Unable to locate the asset without a valid Type');
         return;
     }
     var tenantResources = tenantApi.createTenantAwareAppResources(ctx.session);
     var appManager = tenantResources.appManager;
-    var page = ui.buildPage(ctx.session,ctx.request);
+    var page = ui.buildPage(ctx.session,ctx.request, tenantId);
```

Comment:
@sinthuja Please find the re-factored code at b215dcac00050c70cac7fe85b1c57c99295c5b97
Thanks
Malintha